anybox/recipe/odoo/runtime/patch_odoo.py

```diff
@@ -12,10 +12,9 @@ def do_patch(gevent_script_path):
     isolated from the actual process management logic in the original.
     """
 
-    try:
-        from odoo.service.server import PreforkServer, stripped_sys_argv
-        from odoo.release import version_info
-    except ImportError:
+    from odoo.service.server import PreforkServer, stripped_sys_argv
+    from odoo.release import version_info
+    if False:
```

Comment:
This import does not support Odoo < 10.0 (just encounter the problem on 9.0). You think we should merge anyway?